data/human/news.txt

```diff
@@ -1990,6 +2093,75 @@ news "conspiracy theorist"
 
 
 
+news "players crew member"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	to show
+		"flagship crew" > 30
+	name
+		word
+			"Some crew member"
+	message
+		word
+			`"`
+		word
+			"I'm serving onboard the '<ship>', but I've never met the captain. Can you imagine that?"
+			"Hey captain, good to see you here. I'm looking forward to the next trip (and I'm looking forward to my 100 credits a day)."
+			"You look like a poor excuse for a captain. I'm serving onboard the '<ship>' and I'm sure that my captain is far better dressed than you."
+		word
+			`"`
+
+
+
+news "sticker on ship"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Sticker on ship"
```

Comment:
Is there a way to imply bumper sticker in a funny way?

Reply:
'Sticker on the back of a ship', 'Thruster sticker' or simply 'Bumper Sticker' come to mind. But I feel that none of them really work in this context.

Reply:
The first two actually work for messages, I added them. :smiley: